charts/helm-controller-stack/templates/sourceController/manager_role.yaml

```diff
@@ -0,0 +1,171 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: {{ include "helm-controller-stack.fullname" . }}-manager-role
```

Comment:
Both helmController and sourceController ClusterRole have the same name (`{{ include "helm-controller-stack.fullname" . }}-manager-role`)